crates/compiler/gen_dev/src/lib.rs

```diff
@@ -161,6 +173,51 @@ trait Backend<'a> {
             .starts_with(ModuleName::APP)
     }
 
+    fn list_argument(&mut self, list_layout: InLayout<'a>) -> ListArgument<'a> {
+        let element_layout = match self.interner().get(list_layout) {
+            Layout::Builtin(Builtin::List(e)) => e,
+            _ => unreachable!(),
+        };
+
+        let (element_width_int, alignment_int) =
+            self.interner().stack_size_and_alignment(element_layout);
+
+        let alignment = self.debug_symbol("alignment");
+        self.load_literal_i32(&alignment, Ord::max(alignment_int, 8) as i32);
+
+        let element_width = self.debug_symbol("element_width");
+        self.load_literal_i64(&element_width, element_width_int as i64);
```

Comment:
we are creating some symbols here and in some other places. do these need to be free'd explicitly?

Reply:
I would assume so. They symbol won't have any information about when it should be released. Also, since we always reuse the same names, these will get overwritten if we have multiple list arguments back to back before use. Unless debug symbol does something to make it unique. I don't recall.

This could be related to some of the bugs?

Reply:
it will be unique at least. I'll fix this later then.